### DISCUSSION TITLE
2.8.0 /etc/rancher/k3s/config.yaml': No such file or directory

### DISCUSSION THREAD

**Discussion by HendrikLevering** [mysticaltech/terraform-hcloud-kube-hetzner#1029] [Q&A] [answered]
I created a new 2.8.0 cluster with a different cluster_name than k3s. I just noticed this log during the cluster creation:

module.k8_cluster.module.kube-hetzner.null_resource.control_plane_config["0-1-control-plane-fsn1"] (remote-exec): cp: cannot stat '/etc/rancher/k3s/config.yaml': No such file or directory

The creation completed successfully and the cluster is running. 
My question is, if it is normal that there is no k3 config.yaml? Or maybe this is a bug because the cluster_name is different from k3s.

**Comment by Silvest89** [comment, marked as answer]
@HendrikLevering 
You can ignore that one. For a first run there is no existing config.yaml to backup. I have opened a PR to fix so that it can skip that part on a first run :)

**Reply by HendrikLevering** [reply]
thanks for the explanation :)
